areaOfInterest/areas-of-interest.ts

```diff
@@ -0,0 +1,103 @@
+                //if thumbnail pictures disabled show list of areas as black buttons
+                if (noPic) {
+                    zoneList +=
+                        `<li class="rv-basemap-list-item" style="height: 50px;">
+                                      <div style="position: relative; display: flex; align-items: center; justify-content: center; overflow: hidden; height: 100%">
+                                          <div style="max-height: 50px; max-width: 350px; text-align: center;"></div>
+                                          <div class="rv-basemap-footer"><span>${zone.title}</span></div>
+                                              <button class="rv-body-button rv-button-square md-button md-ink-ripple"
+                                                  type="button" aria-label="bookmark" onclick="setExtent(${zone.xmin}, ${zone.xmax}, ${zone.ymin}, ${zone.ymax}, ${map.id})">
+                                                  <div class="md-ripple-container" style=""></div>
+                                              </button>
+                                          </div>
+                                  </li>`;
+                }
+                //else show list of areas with thumbnails
+                else {
+                    zoneList +=
+                        `<li class="rv-basemap-list-item" style="height: 175px;">
```

Comment:
Move this html into a separate file and import it

Reply:
done!